java-checks/src/main/java/org/sonar/java/checks/unused/UnusedTestRuleCheck.java

```diff
@@ -47,11 +48,16 @@ public List<Tree.Kind> nodesToVisit() {
   @Override
   public void visitNode(Tree tree) {
     ClassTree classTree = (ClassTree) tree;
+    boolean isAbstract = classTree.modifiers().modifiers().stream().anyMatch(m -> Modifier.ABSTRACT == m.modifier());
```

Comment:
We can simplify it using the ModifiersUtils:
`boolean isAbstract =  ModifiersUtils.hasModifier(classTree.modifiers(), Modifier.ABSTRACT);`